tests/install/openqa_webui.pm

```diff
@@ -28,6 +28,7 @@ sub install_from_pkgs {
     assert_script_run "/usr/share/openqa/script/configure-web-proxy $proxy_args";
     if (check_var('OPENQA_WEB_PROXY', 'nginx')) {
         assert_script_run 'systemctl disable --now apache2';
+        assert_script_run 'sed -i "/server_name/ s/localhost/_/" /etc/nginx/nginx.conf';
```

Comment:
I guess this can go into configure-web-proxy, see L28

Shouldn't https://github.com/os-autoinst/openQA/blob/master/script/configure-web-proxy#L49 already do that? Maybe just this regex broke? @asdil12 introduced that initially with https://github.com/os-autoinst/openQA/commit/4fd97ce7237a45bbd83af7d874f54447903fbc38#diff-971564c2eee0f87443db77101fd84c3eb432ce45547d44178d1f1cae179ab83aR24-R33